### DISCUSSION TITLE
GitHub Copilot not show

### DISCUSSION THREAD

**Discussion by klren0312** [community/community#11530] [Copilot Conversations]
GitHub Copilot v1.7.5003
```
vscode info
Version: 1.65.0-insider (user setup)
Commit: 50089c3f92c17584a4aca179f51f220b56c22020
Date: 2022-02-14T05:17:14.990Z
Electron: 16.0.8
Chromium: 96.0.4664.110
Node.js: 16.9.1
V8: 9.6.180.21-electron.0
OS: Windows_NT x64 10.0.18363
```

'configuration.icons' keys represent the icon id and can only contain letter, digits and minuses. They need to consist of at least two segments in the form `component-iconname`.

![image](https://user-images.githubusercontent.com/10903843/154001245-0796dcef-0243-4cf0-9f6f-8cc9b7dd61d1.png)

**Comment by AbhavThakur** [comment]
"icons": {
			"copilot-logo": {
				"description": "GitHub Copilot Icon",
				"default": {
					"fontPath": "assets/copilot.woff",
					"fontCharacter": "\\0041"
				}
			},
			"copilot-warning": {
				"description": "GitHub Copilot Warning",
				"default": {
					"fontPath": "assets/copilot.woff",
					"fontCharacter": "\\0042"
				}
			},
			"copilot-notconnected": {
				"description": "GitHub Copilot Not Connected",
				"default": {
					"fontPath": "assets/copilot.woff",
					"fontCharacter": "\\0043"
				}
			}
		},

**Comment by klren0312** [comment]
The icon appears after the extension  updated
